front/src/applications/operationalStudies/components/MacroEditor/osrdToNge.ts

```diff
@@ -310,21 +276,17 @@ export const loadAndIndexNge = async (
       }
     });
 
-  // Enhance nodes by calling the search API
-  const searchResults = await executeSearch(state, timetableItems, dispatch);
-  searchResults.forEach((searchResult) => {
-    const macroNode: Pick<NodeIndexed, 'full_name' | 'trigram' | 'geocoord'> = {
-      full_name: searchResult.name,
-      trigram: searchResult.trigram + (searchResult.ch ? `/${searchResult.ch}` : ''),
-      geocoord: {
-        lng: searchResult.geographic.coordinates[0],
-        lat: searchResult.geographic.coordinates[1],
-      },
-    };
-    MacroEditorState.getPathKeys(searchResult).forEach((pathKey) => {
-      state.updateNodeDataByKey(pathKey, macroNode);
-    });
-  });
+  const pathOps = timetableItems.flatMap((timetableItem) => timetableItem.pathOps).flat();
+  for (const op of pathOps) {
+    const { trigram, ch } = op.extensions?.sncf ?? {};
```

Comment:
nit : since we spread for `trigram` and `ch`, we could also do it for name ?
```suggestion
    const { trigram, ch } = op.extensions?.sncf ?? {};
    const { name } = op.extensions?.identifier ?? {};
``

Reply:
Since we're only extracting a single field, only using `op.extensions?.identifier?.name` once in a spot where we have plenty of space, I thought it wouldn't improve readability and chose to not use destructuring. If you think that'd be better still, I don't mind, let me know and I can change it.

Reply:
No that's fine :)